src/test/rustdoc/check-source-code-urls-to-def-std.rs

```diff
@@ -0,0 +1,17 @@
+// compile-flags: -Zunstable-options --generate-link-to-definition
+
+#![crate_name = "foo"]
+
+// @has 'src/foo/check-source-code-urls-to-def-std.rs.html'
+
+fn babar() {}
+
+// @has - '//a[@href="https://doc.rust-lang.org/nightly/std/primitive.u32.html"]' 'u32'
+// @has - '//a[@href="https://doc.rust-lang.org/nightly/std/primitive.str.html"]' 'str'
+// @has - '//a[@href="https://doc.rust-lang.org/nightly/std/primitive.bool.html"]' 'bool'
```

Comment:
These will break on beta.
```suggestion
// @has - '//a[@href="{{channel}}/std/primitive.u32.html"]' 'u32'
// @has - '//a[@href="{{channel}}/std/primitive.str.html"]' 'str'
// @has - '//a[@href="{{channel}}/nightly/std/primitive.bool.html"]' 'bool'
```

Reply:
I didn't even know about `{{channel}}`, thanks!

Reply:
I just added it a few months ago: https://github.com/rust-lang/rust/pull/85990